collect_app/src/main/java/org/odk/collect/android/mainmenu/MainMenuActivity.kt

```diff
@@ -329,12 +329,16 @@ class MainMenuActivity : LocalizedActivity() {
 
         val formSavedSnackbarDetails = mainMenuViewModel.getFormSavedSnackbarDetails(uri)
 
-        formSavedSnackbarDetails?.let { it ->
+        formSavedSnackbarDetails?.let {
             SnackbarUtils.showLongSnackbar(
                 binding.root,
                 getString(it.first),
-                action = SnackbarUtils.Action(getString(it.second)) {
-                    formLauncher.launch(FormFillingIntentFactory.editInstanceIntent(this, uri))
+                action = if (it.second == null) {
```

Comment:
This change doesn't have a test. Would probably be good to add one in `MainMenuActivityTest`.